pkgs/tools/games/slipstream/wrapper.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+_base="$HOME/.local/share/slipstream"
```

Comment:
I would rather use `$XDG_DATA_HOME` instead of `$HOME` to not override user's configuration as it could be elsewhere.
```suggestion
_base="$XDG_DATA_HOME/slipstream"
```

Reply:
Good point :) is it reliable enough to depend on XDG_DATA_HOME or should I perhaps write some fallback logic?

Reply:
I think it is reliable enough because it requires a desktop environment if I understand correctly, maybe it could be asserted but I don't know how right now ;)